[Doc][Sphinx] Use imgmath with sphinx >= 1.4, as pngmath has been removed in 1.8

--- doc/sphinx/conf.py.in
+++ doc/sphinx/conf.py.in
@@ -19,13 +19,14 @@ import sys, os
 sys.path = [os.path.abspath('@CMAKE_BINARY_DIR@/src')]+sys.path
 sys.path = [os.path.abspath('@CMAKE_SOURCE_DIR@/src')]+sys.path
 
-import dynamic_graph.tutorial as dgt
+from sphinx import version_info as sphinx_version
 
 # -- General configuration -----------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.pngmath']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo']
+extensions.append('sphinx.ext.pngmath' if sphinx_version < (1, 4) else 'sphinx.ext.imgmath')
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']

